### PR TITLE
[feature] command to list component paths

### PR DESCRIPTION
**Pull request by ryanking**
Add a command to list all configured component paths.

### Test Plan
* unit tests

### References
*

### CODE REVIEW

File: cmd/components_paths.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	componentsCmd.AddCommand(componentsListCmd)
+	componentsListCmd.Flags().StringP("config", "c", "fogg.yml", "Use this to override the fogg config file.")
+}
+
+var componentsListCmd = &cobra.Command{
+	Use:           "paths",
+	Short:         "List paths for all configured components.",
+	SilenceErrors: true, // If we don't silence here, cobra will print them. But we want to do that in cmd/root.go
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var e error
```
Comment (edulop91) [inline cmd/components_paths.go:24]:
```suggestion
```
should this for consistency with the other var definitions?

File: cmd/components_paths.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+		// Set up fs
+		pwd, e := os.Getwd()
```
Comment (edulop91) [inline cmd/components_paths.go:26]:
```suggestion
		pwd, err := os.Getwd()
```
I think `err` is more "idiomatic" + consistent with our go code

File: cmd/components_paths.go [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+		for _, a := range p.Accounts {
+			fmt.Printf("terraform/%s\n", a.AccountName)
+		}
+
+		for _, e := range p.Envs {
+			for _, c := range e.Components {
+				fmt.Printf("terraform/%s/%s\n", e.Env, c.Component)
+			}
+		}
```
Comment (edulop91) [inline cmd/components_paths.go:61]: I wonder if this should be a method on the plan?

**Review by edulop91 — APPROVED** [review]

File: cmd/components_paths.go (hunk 3 above)
Comment (ryanking) [reply cmd/components_paths.go:61]: Yeah I was thinking about that, but given that so far we only need it in one place, I decided to YAGNI for now.

**Review by edulop91 — APPROVED** [review]

**Pull request merged**